installer/cmd/sc/main.go

```diff
@@ -47,6 +47,17 @@ in a Kubernets Cluster.`,
 		// turn off the usage by default on any error
 		SilenceUsage: true,
 	}
+
+	advanced := &cobra.Command{
+		Use:   "advanced",
+		Short: "Set of advanced commands",
+		Long:  "Set of advanced commands for expert users",
+	}
+
+	advanced.AddCommand(
+		cmd.NewCreateGCPBrokerCmd(),
+	)
+
```

Comment:
Do you plan to move all the commands initialization methods here? All of the rest are under pkg folder.

Reply:
no, I am trying to keep the individual commands in the pkg folder. In here, all we do is group the commands under the root "sc" or the advanced "advanced" groups.

Because the root command is created on line 39, this felt like a good place to put this as well.